pkg/installer/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Dockerfile to build installer img initrd
+FROM lfedge/eve-alpine:c114cf1d3ea51534f061f9aa949beb6ac5c12fb3 AS build
```

Comment:
We use 82df60e43ab9f8c935584b8c7b4d0a4b0271d608 now

Reply:
I will bump.

Reply:
Never mind, I will take yours. If they have to be update, we will do them all.